### PR TITLE
White modal

### PR DESCRIPTION
**Pull request by rtluu**
Adds White Modal

### CODE REVIEW

File: src/modules/search/components/Search.tsx [hunk 1]
```diff
@@ -22,16 +22,19 @@ export default function Search({query, setSearch, results, enqueue, isOpen, togg
       isOpen={isOpen}
       onRequestClose={() => toggleSearch(false)}
       style={{
-        overlay: {backgroundColor: 'rgba(20, 20, 20, 0.85)'},
+        overlay: {backgroundColor: 'rgba(20, 20, 20, 0.5)', zIndex: '3'},
```
Comment (jamesmcnamara) [inline src/modules/search/components/Search.tsx]: Use the `zindex.js` function to calculate this

Comment (rtluu) [reply src/modules/search/components/Search.tsx]: I have pulled it down and been trying to write it with the right syntax but can't seem to get it to work. What's the appropriate way to declare the z-index here?

Comment (jamesmcnamara) [reply src/modules/search/components/Search.tsx]: Just go to the `zindex.js` file, and add `'search'` somewhere to the list, and then here do, `zIndex: zindex('search')`

Comment (rtluu) [reply src/modules/search/components/Search.tsx]: Awesome, just fixed this! Thanks for the help, I'm going to merge this now

**Pull request merged**